workflows/sarek.nf

```diff
@@ -554,6 +556,7 @@ workflow SAREK {
 
         // ch_cram_no_markduplicates_restart = Channel.empty()
         cram_markduplicates_no_spark = Channel.empty()
+        cram_sentieon_dedup_no_spark = Channel.empty()
```

Comment:
Sentieon should not go into spark, so I think we can drop that part
```suggestion
        cram_sentieon_dedup = Channel.empty()
```

Reply:
ok